client/src/components/dashboard/Dashboard.js

```diff
@@ -1,15 +1,179 @@
+import { createWebsite } from '../../actions/website';
+
+import { Link } from 'react-router-dom';
 
+/** icons **/
+import { BsPencilSquare, BsServer, BsDisplayFill, BsChevronRight, BsChevronLeft, BsCheck } from 'react-icons/bs'
+/** css **/
+import './Dashboard.css'
+// import DashboardModal from './DashboardModal'
+import homeTemplateImg1 from '../../images/home_page_template1.jpg'
+import homeTemplateImg2 from '../../images/home_page_template2.jpg'
 
 const Dashboard = () => {
+
+    const [show1, setShow1] = useState(false);
+    const [show2, setShow2] = useState(false);
+    const [show3, setShow3] = useState(false);
+
+    const handleCloseOne = () => setShow1(false);
+    const handleCloseTwo = () => setShow2(false);
+    const handleCloseThree = () => setShow3(false);
+
+    const handleShowOne = () => setShow1(true);
+    const handleShowTwo = () => setShow2(true);
+    const handleShowThree = () => setShow3(true);
+
+
+    const dispatch = useDispatch();
+    const websiteIsCreated = useSelector(state => state.website.isCreated);
+
     return (
-        <main>                
-            <Container fluid>
-                You are currently viewing dashboard page. 
-            </Container>
-        </main>
+
+        <Container fluid className="researchify-dashboard-container">
+            <Card className="text-center researchify-dashboard-card">
+
+                <Card.Body>
+                    <Button onClick={handleShowOne}>
+                        {websiteIsCreated ? "Edit the Website" : "Build a new Website"}
+                    </Button>
+                </Card.Body>
+
+                <Card.Body className="researchify-dashboard-card-description">
+                    Click the button to enter Personal Acess Token and get started.
+                </Card.Body>
+
+                {/* Bottom layer of the three icons */}
+                <CardGroup className="researchify-dashboard-card-group">
+                    <Card>
+                        <Link className="researchify-dashboard-card-link">
+                            <Card.Body>
+                                <BsPencilSquare className="researchify-dashboard-card-icons" />
+                            </Card.Body>
+                            <p>Editor</p>
+                        </Link>
+                    </Card>
+                    <Card>
+                        <Link className="researchify-dashboard-card-link">
+                            <Card.Body>
+                                <BsServer className="researchify-dashboard-card-icons" />
+                            </Card.Body>
+                            <p>API Acess Manager</p>
+                        </Link>
+                    </Card>
+                    <Card>
+                        <Link className="researchify-dashboard-card-link">
+                            <Card.Body>
+                                <BsDisplayFill className="researchify-dashboard-card-icons" />

```

Comment:
Maybe try for a different icon for the bs server, and make these icon sizes the same, looks stretched

Reply:
Noted. yea i think might need to discuss later (after the demo?) on the choice of icon cause the `react-icon/bs` doesn't have much icons in it